wfl/calculators/generic.py

```diff
@@ -13,6 +13,10 @@
 def run_autopara_wrappable(atoms, calculator, properties=None, output_prefix='_auto_', verbose=False, raise_calc_exceptions=False):
     """evaluates configs using an arbitrary calculator and store results in SinglePointCalculator
 
+    Defaults to wfl_num_inputs_per_python_subprocess=10, to avoid recreating the calculator for
```

Comment:
Can this be overwritten in a particular instance? If so, it would be clearer to add `def_`/`default_` to `wfl_generic_num_inputs_per_python_subprocess`?

Reply:
It can, I believe, although it might be a good idea to test, given our problems with the default interface.  I guess I do agree that it's worth the even more verbose name.  I'll update that, and try to make a test of that functionality.  I think I'll have to do something like capture stdout and make sure the correct message is printed.

Reply:
please resolve this conversation if you think my latest commit addresses the issue. If not, let's discuss further.